Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -52,8 +55,6 @@ MemoryWidget::MemoryWidget(QWidget* parent) : QDockWidget(parent)
 
   connect(&Settings::Instance(), &Settings::EmulationStateChanged, this, &MemoryWidget::Update);
 
-  setHidden(!Settings::Instance().IsCodeVisible() || !Settings::Instance().IsDebugModeEnabled());
```

Comment:
is `Settings::Instance().IsCodeVisible()` correct for MemoryWidget.cpp? `IsMemoryVisible()` is an option in Settings.cpp so I was wondering if this is a bug.

Reply:
It's probably a bug

Reply:
Did some testing and turns out to be a bug. Fixed it.